requirements.txt

```diff
@@ -24,6 +24,11 @@ texttable==1.6.3
 tornado==6.1
 typing_inspect==0.7.1
 
+# The module set tests install the pytest-inmanta-extensions package.
+# This version constraint ensures that the dependencies of the
+# pytest-inmanta-extensions package remain compatible with inmanta-core.
+inmanta-dev-dependencies==1.42.0
```

Comment:
Will this work for ISO3? (it is currently much lower)

Reply:
This PR will result in a version conflict on ISO3 and ISO4. The goal here is to get approval on the concept, like we usually do. I will create separate pull requests for the different branches.